IO/Ex1.hpp

```diff
@@ -0,0 +1,34 @@
+
+#pragma once
+
+#include <fstream> 
+#include <string>
+#include <vector>
+
+
+class FileHandler {
+public:
+    explicit FileHandler(const std::string path, std::ios_base::openmode permissions); //Ctor
+    FileHandler(FileHandler &other); //CCtor non-const because that tellg/p are modifying 
+    FileHandler(FileHandler &&other) noexcept; //MCtor
+
+    FileHandler& operator=(FileHandler &other); //Copy Assignment non-const because that tellg/p are modifying
```

Comment:
Two spaces before in-line comments and one space between `//` and the comment itself